src/graphql/Offer/Mutations/editOffer.ts

```diff
@@ -36,7 +36,7 @@ export const editOffer = {
   },
   resolve: async (
     _: undefined,
-    attributes: IUpdateOffer,
+    attributes: Omit<IUpdateOffer, "companyUuid">,
```

Comment:
Me fijaría si no es mejor sacarle el `companyUuid` a esa interfaz. Ya que para editar, con tener el uuid, es suficiente para identificar a la oferta.

Reply:
Esto lo cambie porque me saltaba el error pero como ven no va con el pr.
Pero ahora que lo veo mejor hay un problema en la definicion de los tipos, se repiten in IOffer y ICreateOffer al pedo. Despues lo reviso bien
```ts
export interface ICreateOffer {
  title: string;
  description: string;
  hoursPerDay: number;
  minimumSalary: number;
  maximumSalary: number;
  sections?: IOfferSection[];
  careers?: IOfferCareer[];
  companyUuid: string;
}

export interface IUpdateOffer extends ICreateOffer {
  uuid: string;
}

export interface IOfferAttributes extends ICreateOffer {
  companyUuid: string;
}

export interface IOffer extends IUpdateOffer {
  companyUuid: string;
}
```

Reply:
lo anotaría en jira asi no nos olvidamos